glean-core/ios/GleanTests/Config/ConfigurationTests.swift

```diff
@@ -0,0 +1,59 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+@testable import Glean
+
+class ConfigurationTests: XCTestCase {
+    private var config: Configuration?
+
+    override func setUp() {
+        config = Configuration()
+    }
+
+    override func tearDown() {
+        config = nil
+    }
+
+    func testInit() {
```

Comment:
Hm, these tests seem a bit superfluous. Just checking that something is a default value seems ... unnecessary to me.

Reply:
I was dusting the rust off of my testing knowledge with iOS, and we actually test for these things on the Android side, which is what I was looking to for inspiration on what to test. I agree that it seems a little silly to test defaults, and I'm happy to nuke this test if we agree there is no need for it.

Reply:
Let's have them for now, and remove when we feel like it.